github-releases.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+
+ SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+import fetch from 'node-fetch';
+import yaml from "yaml";
+import fs from "fs/promises";
+
+interface GitHubReleaseEntry {
+  id: string
+  name: string
+  tag_name: string
+  html_url: string
+  tarball_url: string
+  body: string
+  assets: {
+    name: string
+    size: number,
+    browser_download_url: string
+  }[]
+  created_at: string
+  prerelease: boolean
+  draft: boolean
+}
+
+export const githubReleases = async () => {
+  fetch('https://api.github.com/repos/hedgedoc/hedgedoc/releases')
```

Comment:
This `fetch` doesn't have a `.catch(err => ...)`.

Reply:
fixed